### PR TITLE
S5 Formalising Design Elements

### PR DESCRIPTION
**Pull request by DarkPurple141**
Wouldn't mind merging this even though not fully done on HP stuff.

Partially Resolving #162, #157.

Includes:

* Footer
* Design Page
* Refactor of :root text types
* Other Style changes

### CODE REVIEW

File: frontend/src/views/Design.vue [hunk 1]
```diff
@@ -0,0 +1,83 @@
+<template>
```
Comment (NunoDasNeves) [inline frontend/src/views/Design.vue]: this page is great

File: frontend/src/views/Home.vue [hunk 2]
```diff
@@ -1,8 +1,45 @@
 <template>
-  <div class="main-content home">
-    <AppLogo :first="'Smart'" :last="'Course'"/>
-    <p>Making the most of your degree.</p>
-    <Search/>
+  <div>
+    <section class="main-content home">
```
Comment (NunoDasNeves) [inline frontend/src/views/Home.vue]: this page lookin' good also

**Comment by DarkPurple141** [comment]
I'll leave this for another hour just in case the fellas want to have a looksie.

File: frontend/src/views/Home.vue (hunk 2 above)
Comment (DarkPurple141) [reply frontend/src/views/Home.vue]: Definitely a WIP. Need to play with mobile sizing, also not certain on card design for info panels.

**Comment by DarkPurple141** [comment]
Can't reply directrly to your review @NunoDasNeves:

But yeah that's the plan. Been trying to take components away from setting font-sizes colours directly, but by no means comprehensive.

**Pull request merged**